src/main/java/jsmith/nknsdk/network/ClientTunnel.java

```diff
@@ -249,18 +249,19 @@ private boolean setupWsConnection() {
                         ws.sendPacket(receiptMsg);
                         LOG.debug("Sending receipt msg");
                     }
-
-                    switch (payload.getType()) {
-                        case ACK:
-                        case TEXT:
-                        case BINARY:
-                            cm.onInboundMessage(from, payload);
-                            break;
-
-                        default:
-                            LOG.warn("Got invalid payload type {}, ignoring", payload.getType());
-                            break;
-                    }
+                    cm.onInboundMessage(from, pldMsg);
+
+//                    switch (payload.getType()) {
```

Comment:
We removing this part of logic?

Reply:
Hm, thats fair. Before encryption there was different handling of ACK, iirc.
However, it is better to just delete it, don't comment out bits of code (don't keep dead code in the codebase. We have git if we ever need to see older version)